### ISSUE TITLE
Digital Marin: news list filter label

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1787]
@aschmoe @lukefretwell here's another one from Marin that was submitted that doesn't seem urgent but worth addressing:

The "Category" on the News page filter section should not be in a <label> element since it is not a label for a particular input. Instead, this is a grouping and the proper HTML for this is to put it inside of <legend> which is inside of a <fieldset> element and all of the checkboxes should be inside of this fieldset. This is important for accessibility as this will allow the screen reader to properly announce the checkboxes as being part of the Category grouping.

Here is the proper HTML for this named grouping:
<fieldset>
<legend>Category</legend>
<div class="checkbox"...
<div class="checkbox"...
...
</fieldset>

Using semantic HTML will change the look of this filter, but I think it looks even better than the correct styling so I don't think you made to make any CSS changes, at least not for the theme that Digital Marin is using.
